md_link_check_config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ignorePatterns": [
+      {
+        "pattern": "^https://github.com"
+      },
+      {
+        "pattern": "^./"
+      },
+      {
+        "pattern": "^/get_involved$"
```

Comment:
This one is a link that jekyll will turn into a proper link based on the jekyll config. There is probably a nicer way to deal with this, but I am out of time now. Help welcome!